src/native_background.ts

```diff
@@ -270,14 +275,34 @@ export async function getenv(variable: string) {
  You'll get both firefox binary (not necessarily an absolute path) and flags */
 export async function ffargs(): Promise<string[]> {
     // Using ' and + rather that ` because we don't want newlines
-    let output = await pyeval(
-        'handleMessage({"cmd": "run", ' +
-            '"command": "ps -p " + str(os.getppid()) + " -oargs="})["content"]',
-    )
-    return output.content.trim().split(" ")
+    if ((await browserBg.runtime.getPlatformInfo()).os === "win") {
+        return []
+    } else {
+        let output = await pyeval(
+            'handleMessage({"cmd": "run", ' +
+                '"command": "ps -p " + str(os.getppid()) + " -oargs="})["content"]',
+        )
+        return output.content.trim().split(" ")
+    }
 }
 
 export async function getProfileDir() {
+    // Windows users must specify their Firefox profile directory
+    // via 'set profiledir [directory]'. Windows profile directory
+    // path must be properly escaped. For example, a correct way
+    // to set 'profiledir' on Windows is as shown below:
+    //
+    // : set profiledir C:\\Users\\<User-Name>\\AppData\\Roaming\\Mozilla\\Firefox\\Profiles\\8s21wzbh.Default
```

Comment:
I'd much rather we escaped it for them. Where does the requirement come from? I think python has `os.path` for dealing with these things.

Reply:
Thanks! Can we delete this comment then?

Reply:
I've updated the comment [here][0] with further details. 

Note that Vimperator originally used to require escaping paths. So, there might be similar expectations from some users. Currently we are supporting profile directory paths _only_ and _exactly_ as displayed in `about:support`. 

I believe the rationale for this is that it will allows users to quickly copy-paste the profile directory path and set via `:set profiledir`, which I think also makes sense. We just need to communicate it clearly, and stay coherent in  the code-base.

Thanks.

[0]: https://github.com/gsbabil/tridactyl/commit/9ffc161b865d0ac6a5fe3712cc70cc74de3f2a98#diff-9ddd9172e4a54cdd30d56e5f10d5facaR298